### PR TITLE
doc/cephadm: Improve multiple files

### PR DESCRIPTION
**Pull request by anthonyeleven**
## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

**Review by baum — APPROVED** [review]
lgtm 🖖

File: doc/cephadm/operations.rst [hunk 1]
```diff
@@ -483,30 +483,30 @@ flagged as an anomaly and a healthcheck (WARNING) state raised.
...
+health check.
 
 CEPHADM_CHECK_LINKSPEED
 ~~~~~~~~~~~~~~~~~~~~~~~
-This check is similar to the MTU check. Linkspeed consistency is a factor in
-consistent cluster performance, just as the MTU of the NICs on the OSDs is.
-This check determines the linkspeed shared by the majority of OSD hosts, and a
+This check is similar to the MTU check. Link speed consistency is a factor in
+consistent cluster performance, as is the MTU of the OSD node network interfaces.
+This check determines the links peed shared by the majority of OSD hosts, and a
```
Comment (zdover23) [inline doc/cephadm/operations.rst:509]: s/links peed/link speed/

But I enjoy the past tense of the verb "to pee" as much as the next guy.

(I'll take care of this in a force-push now. You don't have to do anything.)

**Comment by zdover23** [comment]
jenkins retest this please

**Review by zdover23 — APPROVED** [review]

File: doc/cephadm/operations.rst (hunk 1 above)
Comment (anthonyeleven) [reply doc/cephadm/operations.rst]: With certain NICs I've suffered ...

Comment (zdover23) [reply doc/cephadm/operations.rst]: It's wild how long it's taking for this PR to pass its tests. I've been waiting up for almost two hours to do the backports on this one. </vent>

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/56129 - Squid backport
https://github.com/ceph/ceph/pull/56130 - Reef backport
https://github.com/ceph/ceph/pull/56131 - Quincy backport

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/56134 - Quincy backport second attempt

**Pull request merged**